pkgs/tools/admin/lxd/default.nix

```diff
@@ -58,6 +59,8 @@ buildGoPackage rec {
     installShellCompletion --bash go/src/github.com/lxc/lxd/scripts/bash/lxd-client
   '';
 
+  passthru.tests.lxd = nixosTests.lxd;
```

Comment:
Out of curiosity, what does it do?

Reply:
It allows third-party tooling like ofborg discover nixos tests and build them.